source/fundamentals/indexes.txt

```diff
@@ -397,6 +397,34 @@ Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB s
 
 .. driver-content-end
 
+Clustered Indexes
+~~~~~~~~~~~~~~~~~
+
+**Clustered indexes** are indexes that improve the performance of
+insert, update, and delete operations on **clustered collections**.
+Clustered collections store documents ordered by the clustered index key
+value.
+
+To create a clustered index, specify a ``ClusteredIndexOptions`` in
+your ``CreateCollectionOptions``. The ``ClusteredIndexOptions`` must
```

Comment:
Issue:
Not sure what "ClusteredIndexOptions" and "CreateCollectionOptions" are because they haven't been introduced yet and aren't trivial to figure out.

Suggestion:
Identify what they are instead of only referring to them by name.